src/test/java/com/uid2/shared/auth/RotatingKeyAclProviderTest.java

```diff
@@ -27,7 +30,7 @@ public class RotatingKeyAclProviderTest {
 
     @Before public void setup() {
         mocks = MockitoAnnotations.openMocks(this);
-        keyAclProvider = new RotatingKeyAclProvider(cloudStorage, "metadata");
+        keyAclProvider = new RotatingKeyAclProvider(cloudStorage, new GlobalScope(new CloudPath("metadata")));
```

Comment:
is it worth adding a separate test for using new SiteScope and also that would test the other overloaded `com.uid2.shared.store.reader.RotatingKeyAclProvider#loadContent()` method (not just `loadContent(JsonObject metadata`) ) for complete coverage?

Reply:
Ok, will add a test with site scope. Will also add one for loadContent().
Honestly, maybe we should get rid of loadContent(metadata), so far I've never seen anything using anything but a.loadContent(a.getMetadata()) which is the exact implementation of the loadContent()

Reply:
Ok, done.